transcoder/src/codec.go

```diff
@@ -115,6 +115,11 @@ func GetMimeCodec(stream *ffprobe.Stream) *string {
 		ret := "alac"
 		return &ret
 
+	// Based on https://cconcolato.github.io/media-mime-support/#video/mp4;%20codecs=%22dtsc%22
+	case "dts":
+		ret := "dtsc"
```

Comment:
I don't think your source is correct, example dts mime type computation from [exoplayer](https://github.com/google/ExoPlayer/blob/dd430f7053a1a3958deea3ead6a0565150c06bfc/library/common/src/main/java/com/google/android/exoplayer2/util/MimeTypes.java#L439-L446)

it looks like this is a more complicated mime format that require profiles & co embedded (ex dtsx is `/vnd.dts.uhd;profile=p2`)

Reply:
Is this good to go then or is there more action required on my part?

Reply:
i think you need to detect which variant of dts the file is in & set `ret` to either `dtsl`,`dtsx`, dtsh` & co accordingly

Reply:
Pushed a fix for this. Running a build now and will test to see if it works